packaging/torchvision/meta.yaml

```diff
@@ -44,7 +50,9 @@ test:
   requires:
     - pytest
     - scipy
-    - av
+    # AV is disabled temporarily until the jpeg/jpeg-turbo conflicts are solved
+    # in conda-forge
+    # - av
```

Comment:
This is unfortunate because we stop testing our video reading functionality (all tests in `test_io` are skipped).

If we were to use `libjpeg` instead of libjpeg-turbo, would that fix this?

Reply:
I will restore this, as the culprit corresponds to Pillow